src/backend/_scss/application.scss

```diff
@@ -2195,3 +2204,90 @@ a.skip:hover {
   top: 1px;
   color: $black;
 }
+
+// Filters for mobile view
+
+.dropdown-inner {
+  max-height: 200px;
+  max-height: 60vh;
```

Comment:
I am trying to do progressive enhancement here. If the browser does not support "vh" then I think it will just use the previously declared pixel value "200px".